ui/desktop/openapi.json

```diff
@@ -3590,7 +3590,8 @@
             "nullable": true
           },
           "created_at": {
-            "type": "string"
+            "type": "string",
+            "format": "date-time"
```

Comment:
does this not require client side changes

Reply:
I think this is just an openapi annotation. If it changed the generated code we'd see that in this diff, but it seems that it does not: https://github.com/block/goose/blob/12f7ef5900be42a48445e757bc2a444a193f6b86/ui/desktop/src/api/types.gen.ts#L712-L713

Reply:
turns out it's a plugin: https://heyapi.dev/openapi-ts/plugins/transformers#dates